### ISSUE TITLE
Create card management page (Druckerei) on Self Service Portal

### ISSUE THREAD

**Issue by seluianova** [digitalfabrik/entitlementcard#1423]
**Is your feature request related to a problem? Please describe.**
The card management system (“Digital Print Shop”) should allow users to generate PDF files containing an activation code for the KoblenzPass app and a static QR code for identification (hybrid option).

- Only PDF files/passes authorized/provided through the user data interface can be created
- As with the self-service portal, user must enter the correct data to create the PDF file
- The card management system does not allow to invalidate cards

When a card is created from the print shop (Druckerei), an entry must also be created in the assignment table (Zuweisungstabelle) with the UserHash so that these cards can also be deleted via the authorization table (Berechtigungstabelle).

**Additional context**
TBD?

blocked by #1418

**Comment by seluianova** [comment]
@f1sh1918 is it a duplicate of https://github.com/digitalfabrik/entitlementcard/issues/1419?

**Comment by f1sh1918** [comment]
> @f1sh1918 is it a duplicate of https://github.com/digitalfabrik/entitlementcard/issues/1419?

@seluianova 
Yes it looks like this issue can be closed. 
In the other one the assignment table between cards and user data is missing but there is also a separate issue for that
